lib/bob_dns.ml

```diff
@@ -63,7 +63,12 @@ module Transport :
                      (msgf "Error %s connecting to nameserver %a:%d"
                         (Unix.error_message errno) Ipaddr.pp ip port))
             | Ok _ as v -> Fiber.return v)
-          (fun () -> Fiber.wait cancel >>| fun () -> Error (msgf "Cancelled"))
+          (fun () ->
+            Fiber.wait cancel >>| fun () ->
+            Log.warn (fun m ->
```

Comment:
It was at line 101 (106) I meant it would be nice to have a warning. Where we are asked to cancel something which we can't cancel.